src/value/borrowed.rs

```diff
@@ -59,8 +67,54 @@ impl<'val> BorrowedSymbolToken<'val> {
             source,
         }
     }
+
+    /// Decorates the [`BorrowedSymbolToken`] with text.
+    pub fn with_text(mut self, text: &'val str) -> Self {
+        self.text = Some(text);
+        self
+    }
+
+    /// Decorates the [`BorrowedSymbolToken`] with a local ID.
+    pub fn with_local_sid(mut self, local_sid: SymbolId) -> Self {
+        self.local_sid = Some(local_sid);
+        self
+    }
+
```

Comment:
Codecov indicates that this was not tested--is it correct?  Might be worth re-jiggering some test cases to hit this (probably true for `OwnedSymbolToken` tests as well).

Reply:
I do not think these lines that codecov shows as not tested are correct. The tests seems to cover these lines.